src/Tribe/Views/V2/Views/Widgets/Widget_List_View.php

```diff
@@ -85,12 +86,54 @@ protected function setup_template_vars() {
 		$template_vars = parent::setup_template_vars();
 
 		// Here update, add and remove from the default template vars.
-		$template_vars['view_more_link']   = tribe_get_events_link();
-		$template_vars['widget_title']     = $this->context->get( 'widget_title' );
-		$template_vars['show_latest_past'] = false;
+		$template_vars['view_more_link']     = tribe_get_events_link();
+		$template_vars['widget_title']       = $this->context->get( 'widget_title' );
+		$template_vars['no_upcoming_events'] = $this->context->get( 'no_upcoming_events' );
+		$template_vars['show_latest_past']   = false;
 		// Display is modified with filters in Pro.
 		$template_vars['display'] = [];
 
 		return $template_vars;
 	}
+
+	/**
```

Comment:
:no_entry_sign: Doc comment for parameter "$events" missing

<sub>hash: 8409de11121f657b74ca8fa4e01cdd8b</sub>